docs/reference/errors-and-warnings/NU5128.md

```diff
@@ -0,0 +1,80 @@
+---
+title: NuGet Warning NU5125
+description: NU5125 Warning code
+author: zivkan
+ms.date: 09/06/2019
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords: 
+  - NU5128
+---
+
+# NuGet Warning NU5128
+
+## Scenario 1
+
+<pre>Some target frameworks declared in the dependencies group of the nuspec and the lib/ref folder do not have exact matches in the other location. Consult the list of actions below:
+- Add a dependency group for .NETStandard2.0 to the nuspec</pre>
+
+### Issue
+
+The `lib/<tfm>/` or `ref/<tfm>/` directory in the package contains at least one file for the Target Framework Moniker (TFM) specified in the warning message. However, no dependency group exists for this TFM in the `nuspec` file. This may cause package consumers who use dependency information to believe the package is not compatible with the TFM.
+
+### Solution
+
+* Let NuGet automate `lib/` files and dependency groups
+
+If possible, use NuGet's MSBuild pack task, as it automatically matches assembly TFMs with dependency groups.
+
+* Add a new dependency group to the `nuspec` file
+
+If you are using a custom `nuspec` file, we recommend each TFM for which `lib/` or `ref/` assemblies exist should have a matching dependency group, even if the dependencies are the same as the next compatible TFM. For example, if a package contains `netstandard1.0` and `netstandard2.0` assemblies, and the dependencies are the same for both, we recommend both TFMs be listed as dependency groups with duplicate dependency items.
+
+Note that the TFM identifier used in the assembly paths use a different format to the TFM identifier used in dependency groups. The warning message specifies the correct name to use in the dependency group. If your package does not have any dependencies for that target framework, use an empty group. For example:
+
+```xml
+<package>
+  <metadata>
+    ...
+    <dependencies>
+      <group targetFramework=".NETFramework4.7.2" />
+    </dependencies>
+  </metadata>
+  ...
+<package>
+```
+
+* Remove the `lib/` or `ref/` files
+
+If you do not wish your package to be compatible with the reported TFM, modify your project such that no `lib/<tfm>/` or `ref/<tfm>/` files are in the package for that TFM.
```

Comment:
Maybe have an actual practical example? 
I think that might be the case in some of the other warnings. 

Meaning you'd show the nuspe and the file directory structure.

Reply:
I added an example, but didn't add a full nuspec or directory structure. Given you're already approved the PR, I guess it's acceptable :)

Reply:
Yeah, it's ok. 
Once I saw the change I liked it and didn't think it's necessary to add more detail.